foundation_cms/static/scss/components/footer.scss

```diff
@@ -156,38 +156,35 @@
   }
 
   &__cta {
-    .site-footer-ns__button--cta {
-      display: flex;
-      align-items: center;
+    .btn-primary {
```

Comment:
Does the donate button on footer share the same styling with `.btn-primary` but only differ in background colour?  If so, I wonder if we should create a variant of `.btn-primary` with just the background color override?